afs/media/js/views/components/workflows/create-project-workflow/add-things-step.js

```diff
@@ -102,6 +106,37 @@ define([
             });
         }, null, "arrayChange");
 
+        const loadExistingCollection = async function(){
+            const projectRelatedResources = await (await window.fetch(`${arches.urls.related_resources}${self.projectResourceId()}`)).json();
+            self.collectionResourceId(projectRelatedResources.related_resources.related_resources.find(x=>x.graph_id==="1b210ef3-b25c-11e9-a037-a4d18cec433a").resourceinstanceid);
+
+            const collectionRelatedResources = await (await window.fetch(`${arches.urls.related_resources}${self.collectionResourceId()}`)).json();
+            const childPhysicalThingsValueIds = [
+                '77d8cf19-ce9c-4e0a-bde1-9148d870e11c', //sample
+                '7375a6fb-0bfb-4bcf-81a3-6180cdd26123', //sample location
+                '31d97bdd-f10f-4a26-958c-69cb5ab69af1', //anlysis area
+            ];
+
+            self.startValue(
+                collectionRelatedResources.related_resources.related_resources
```

Comment:
@njkim We should confirm with @dwuthrich whether sample / analysis area instances should appear in this list or not.